api/src/controllers/publications.js

```diff
@@ -150,6 +154,158 @@ async function readAllPublicationsByTeam(req, res) {
     res.status(200).json(foundPublication);
 }
 
+/**
+ * Given a google scholar user id, this function calls an AWS Lambda Python function that scrapes the publications off the user's profile and returns them.
+ * Some extra processing is done by this function to make the publications returned fit the publication model of the DB.
+ * Endpoint: /publications/import/:id
+ * @param req request object - google scholar user id given in the url
+ * @param res response object - array of publication objects
+ */
+async function getGoogleScholarPublications(req, res) {
+    let author = req.params.id;
+    console.log(author);
+
+    const client = lambda;
+    const params = {
+        FunctionName: 'import-pub-opt',
+        InvocationType: 'RequestResponse',
+        Payload: "{\"author_id\": \""+ author+"\"}"
```

Comment:
may be good to encapsulate this in a constant somewhere; probably a refactor to be considered later.